portal/src/test/scala/cromwell/pipeline/service/UserServiceTest.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.UserRepository
+import cromwell.pipeline.datastorage.dto.user.DeactivateUserRequestByEmail
+import cromwell.pipeline.datastorage.dto.{ User, UserDeactivationByEmailResponse, UserDeactivationByIdResponse, UserId }
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.concurrent.ScalaFutures._
+import org.scalatest.{ Matchers, WordSpec }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class UserServiceTest extends WordSpec with Matchers with MockFactory {
+  implicit val executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
+  private val userRepository: UserRepository = stub[UserRepository]
+  private val userService: UserService = new UserService(userRepository)
+
+  "UserService" when {
+    "deactivateByEmail" should {
+      "return user's email and active value" in {
+        val email = "email"
+        val deactivateUserRequestByEmail = DeactivateUserRequestByEmail(email)
+        val user = User(UserId("123"), email, "hash", "salt", "name", "lastName", active = false)
+
+        (userRepository.deactivateByEmail _ when email).returns(Future(1))
+        (userRepository.getUserByEmail _ when email).returns(Future(Option(user)))
+
+        val emailResponse = UserDeactivationByEmailResponse(email, active = false)
+        whenReady(userService.deactivateByEmail(deactivateUserRequestByEmail)) { result =>
+          result shouldBe Option(emailResponse)
```

Comment:
We use `Option` if we don't know if there is `null` or value.
If we're sure we will have value here -- use `Some()`
If we're sure we will have null here -- use `None`

Reply:
I've fixed it